src/Microsoft.ML.ImageAnalytics/ImageGrayscale.cs

```diff
@@ -32,18 +32,8 @@ namespace Microsoft.ML.Transforms.Image
     // REVIEW: Rewrite as LambdaTransform to simplify.
     // REVIEW: Should it be separate transform or part of ImageResizerTransform?
     /// <summary>
-    /// <see cref="ITransformer"/> produced by fitting the <see cref="IDataView"/> to an <see cref="ImageGrayscalingEstimator" />.
+    /// <see cref="ITransformer"/> resulting from fitting a <see cref="ImageGrayscalingTransformer"/>.
```

Comment:
>a [](start = 58, length = 1)

a --> an

Reply:
I think in past month I learn more about English grammar rather than machine learning or C# development.
Not sure what to think about this.

---
In reply to: [276906392](https://github.com/dotnet/machinelearning/pull/3376#discussion_r276906392) [](ancestors = 276906392)